### ISSUE TITLE
Handle 'sharedApplication' is unavailable: not available on iOS (App Extension)

### ISSUE THREAD

**Issue by bsudekum** [mapbox/mapbox-events-ios#47]
I'm getting an error when using this library inside of an extension:

```
'sharedApplication' is unavailable: not available on iOS (App Extension) - Use view controller based solutions where appropriate instead.
```

This is a pretty widespread and documented issue, [ref](https://miqu.me/blog/2016/11/28/app-extensions-xcode-and-cocoapods-omg/).

/cc @rclee

**Comment by friedbunny** [comment]
This could be addressed alongside #44, as it’s more or less the same codepath.

**Comment by dmiluski** [comment]
Fix merged, should be available in the next release
